rke/templates/templates.go

```diff
@@ -173,9 +174,13 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.21.0-rancher0 <1.21.14-rancher1-1": aciv500,
 			">=1.21.14-rancher1-1 <1.22.0-rancher0": aciv523,
 			">=1.22.0-rancher0 <1.22.11-rancher1-1": aciv500,
-			">=1.22.11-rancher1-1 <1.23.0-rancher0": aciv523,
+			">=1.22.11-rancher1-1 <1.22.13-rancher1-1": aciv523,
+			">=1.22.13-rancher1-1 <1.23.0-rancher0": aciv5233,
 			">=1.23.0-rancher0 <1.23.8-rancher1-1":  aciv500,
-			">=1.23.8-rancher1-1":                   aciv523,
+			">=1.23.8-rancher1-1 <1.23.10-rancher1-1":  aciv523,
+			">=1.23.10-rancher1-1 <1.24.0-rancher0":  aciv5233,
```

Comment:
Same as above, I believe `v1.23.10-rancher1-2` should be used instead.

Reply:
addressed in d4d4d56db8839fea2e11f7318c3708d21b357a8d